examples/glue_benchmark/glue_mrpc.py

```diff
@@ -0,0 +1,193 @@
+# Copyright 2023 The KerasNLP Authors
```

Comment:
Let's rename this to glue.py and take in mrpc as a task name.

Totally ok if we just error out for any other glue tasks right now, but that will allow us to grow this script into one for testing glue in its entirety.

Reply:
Ok, then should I just remove the old `glue.py`? And I am sorry I didn't fully understand `take in mrpc as a task name.` should I include all tasks with `mrpc` being the default or will it only be for `mrpc`?

Reply:
@mattdangerw I think in the long term we will probably only use `stsb` (regression) and `mrpc` (classification). I am fine with naming it as `glue.py`, but we should add some disclaimer at the header saying "Benchmark with [selected tasks]".

@susnato I believe what Matt is suggesting is:
- rename `glue_mrpc.py` to `glue.py`
- set `task` flag, and default to `mrpc`. If other tasks are passed, error out.
- keep code minimal to `mrpc` task, and add comments to note out other tasks are not supported yet.

This leaves room for scalability.